table/block_based/index_reader_common.cc

```diff
@@ -43,12 +43,12 @@ Status BlockBasedTable::IndexReaderCommon::GetOrReadIndexBlock(
     return Status::OK();
   }
 
-  ReadOptions read_options;
+  ReadOptions read_opts = read_options;
```

Comment:
We need to be selective here and only copy rate_limiter_priority. Other options like verify_checksum and async_io could have unintended consequences. Maybe to be safe, just pass rate_limiter_priority explicitly to this function, rather than read_options.

Reply:
will update.